PKHeX.Core/Saves/Access/SaveBlockAccessor8SWSH.cs

```diff
@@ -415,6 +417,11 @@ public SaveBlockAccessor8SWSH(SAV8SWSH sav)
         public const uint KSparringFairyPartySlot1Sweet = 0xB14624FF; // U32 Alcremie Sweet ID if 1st PKM used in party, otherwise -1
         public const uint KSparringFairyPartySlot2Sweet = 0xB046236C; // U32 Alcremie Sweet ID if 2nd PKM used in party, otherwise -1
         public const uint KSparringFairyPartySlot3Sweet = 0xB3462825; // U32 Alcremie Sweet ID if 3rd PKM used in party, otherwise -1
+        
+        public const uint KCobalionFootprintPercentage = 0x4D50B655; // U32 Footprints of Cobalion collected on Crown Tundra; values go from 0-100
```

Comment:
Duplicates?

```cs
1DADFC9D4D50B655	WK_EV_SUB_RIGEL_012_KOBALON
ADB4074C771E4C88	WK_EV_SUB_RIGEL_012_TERAKION
4B2A167DAD67A297	WK_EV_SUB_RIGEL_012_BIRIZION
4FD3175D6106657D	WK_EV_SUB_RIGEL_012_KELDHIO
55CD1EAA31A13425	WK_EV_SUB_RIGEL2_014_TALK_COUNT
```

Reply:
Yeah derped it should be good now. Too sleepy.

g'night